facebook-commerce-pixel-event.php

```diff
@@ -398,14 +398,52 @@ function handle<?php echo $event_name; ?>Event() {
 		 */
 		public static function build_event( $event_name, $params, $method = 'track' ) {
 
-			return sprintf(
-				"/* %s Facebook Integration Event Tracking */\n" .
-				"fbq('%s', '%s', %s);",
-				WC_Facebookcommerce_Utils::getIntegrationName(),
-				esc_js( $method ),
-				esc_js( $event_name ),
-				json_encode( self::build_params( $params, $event_name ), JSON_PRETTY_PRINT | JSON_FORCE_OBJECT )
-			);
+			// do not send the event name in the params
+			if ( isset( $params['event_name'] ) ) {
+
+				$event_name = $params['event_name'];
```

Comment:
Should we be overriding the `$event_name` value that's passed to the method here? This causes the "event_name" test case to fail.

Reply:
You're right, we always pass the event_name as the first param to the inject methods, so we really don't need to override it here.